examples/lock-app/mbed/mbed_app.json

```diff
@@ -11,7 +11,7 @@
             "nsapi.default-wifi-ssid": "\"YOUR_SSID\"",
             "nsapi.default-wifi-password": "\"YOUR_PASSWORD\"",
             "mbed-trace.max-level": "TRACE_LEVEL_DEBUG",
-            "mbed-trace.enable": false
+            "mbed-trace.enable": true
```

Comment:
I think traces shouldn't be enabled by default

Reply:
Aggree, I forgot to clean that up.